.github/CODEOWNERS

```diff
@@ -73,6 +73,7 @@ exporter/lokiexporter/                                   @open-telemetry/collect
 exporter/mezmoexporter/                                  @open-telemetry/collector-contrib-approvers @dashpole @billmeyer @gjanco
 exporter/opencensusexporter/                             @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
 exporter/opensearchexporter/                             @open-telemetry/collector-contrib-approvers @Aneurysm9 @MitchellGale @MaxKsyunz @YANG-DB
+exporter/otelarrowexporter/                              @open-telemetry/collector-contrib-approvers @open-telemetry/arrow-approvers
```

Comment:
I'd much rather that you list members as we now check users membership when they're added as codeowners.

Reply:
OK, I had proposed @jmacd and @gramidt; other candidates are @lquerel and @moh-osman3.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/26491#issuecomment-1710441021 thanks